source/user/sgx.rst

```diff
@@ -27,27 +27,54 @@ Any :attr:`scf.hf.SCF` object :attr:`mf` can be converted to an equivalent objec
 
 .. code::
 
-  converged SCF energy = -76.0267374704185
-  converged SCF energy = -76.0267978618974
-
-In this case, the error of SGX compared to analytical exchange is about 0.06 mEh. The line
+    converged SCF energy = -76.026765673118
+    converged SCF energy = -76.0267374704184
+    converged SCF energy = -76.0267978618973
+    converged SCF energy = -76.0267978618973
+    converged SCF energy = -76.0267978671524
+    /home/kyle/Research/projects/devpyscf/pyscf/pyscf/sgx/sgx.py:126: UserWarning: 
+                        P-junction screening is not compatible with SGX J-matrix.
+                        Setting dfj = True. If you want to use SGX J-matrix,
+                        set pjs = False to turn off P-junction screening.
+      warnings.warn(msg)
+    converged SCF energy = -76.0267978671524
+    converged SCF energy = -76.0267685471926
```

Comment:
This looks a bit sketchy in the docs: you have 7 converged SCF energies and a five-line warning. How is one supposed to interpret this?

Reply:
I was just showing what the output looks like for the example script, but I guess it doesn't give any useful information and adds confusion. The UserWarning is explained below anyways. Should I just not include the output?